### ISSUE TITLE
Development process documentation specifically about contribution

### ISSUE THREAD

**Issue by codecurve** [cellml/libcellml-archived#43]
As a sub-task of #42, this document will describe the process for contributing new code or changes to existing code, as well as contributing new documentation or changes to existing documentation.

**Comment by codecurve** [comment]
Closing issue in favour of the corresponding pull request (#44)

**Comment by codecurve** [comment]
We've decided to keep issues open until they are resolved, rather than closing them when a linked pull request is created, hence this one has been reopened.

**Comment by codecurve** [comment]
Please provide feedback on  #44

**Comment by nickerso** [comment]
Randall, I suspect a more useful comment to make is something like: "I believe #44 addresses this issue and is ready for review". Just to provide an indication that you consider the pull request complete.

**Comment by codecurve** [comment]
True, I do believe #44 addresses this issue and is ready for review, please proceed.
